oesukam/andela-sendit
# #163261083 Enhance the UI/UX of create a parcel page

#### What does this PR do?
Enhance the UI of "Create a parcel page"

#### Description of Task to be completed?
- Add the model component to display errors

#### How should this be manually tested?
- Signup a new account without confirming the email
- Log in with your credential
- Navivate to create a parcel
- Fill and submit the form to see the error model appears
